tuna/example/example_worker.py

```diff
@@ -51,7 +51,7 @@ def set_db_tables(self) -> None:
   def step(self) -> bool:
     """Main functionality of the worker class. It picks up jobs in new state and executes them"""
 
-    if not self.get_job("new", "running", False):
+    if not self.get_job({"new"}, "running", False):
```

Comment:
Curly braces would indicate a dictionary in python. This function hints that the find_state parameter is a list.

Reply:
ive made it a dict because it later translates to mysql nicely for querring for jobs where the state is in (). i can look into changing to a list or something like that